### ISSUE TITLE
Less pager ignores arrow up/down

### ISSUE THREAD

**Issue by luigiberrettini** [microsoft/terminal#6948]
# Environment

```none
Windows build number: Microsoft Windows NT 10.0.19041.0
Windows Terminal version (if applicable): 1.0.1811.0
Windows PowerShell 5.1.19041.1 and PowerShell 7.0.2
```

# Steps to reproduce

 - Run `git diff HEAD~1` from within a git repo folder
 - Press the up arrow
 - Press the down arrow

# Expected behavior

Pressing the up/down arrow the less pager move one line up/down
This is the behavior I experience:
 - in the cmd shell both inside and outside of Windows Terminal
 - in Windows PowerShell outside of Windows Terminal
 - in PowerShell outside of Windows Terminal

# Actual behavior

Pressing the up/down arrow result in nothing
The key bindings listed in the less help still work:
```none
e  ^E  j  ^N  CR  *  Forward  one line   (or N lines).
y  ^Y  k  ^K  ^P  *  Backward one line   (or N lines).
```

**Comment by zadjii-msft** [comment]
I believe there's an issue floating around about input in git bash being weird recently, but I can't find it off the top of my head

**Comment by DHowett** [comment]
Thanks for the report! This is a /dupe of #6859 (the feature that will fix this issue) - Terminal 1.1 isn't susceptible to this, and is available in the Preview channel today.
